api/currency/definition.proto

```diff
@@ -64,4 +66,34 @@ message BurnCurrencyRequest {
     string currency_id = 2;
     /* Amount of currency to burn */
     int64 amount = 3;
-}
\ No newline at end of file
+}
+
+message GetBalanceOfPlayerRequest {
+    string game_title_id = 1;
```

Comment:
oh yea, no need for game_title since it'll come from the auth and no need for "game" in `currency_type_id` just to follow our other conventions